Makefile

```diff
@@ -61,7 +61,11 @@ TEST_CMD := @APIGEE_ACCESS_TOKEN="$(APIGEE_ACCESS_TOKEN)" \
 		-n 4 \
 		--api-name=communications-manager \
 		--proxy-name="$(PROXY_NAME)" \
-		-s
+		-s \
+		--reruns 5 \
+		--reruns-delay 5 \
+		--only-rerun 'AssertionError: Unexpected 429 Timeout'
```

Comment:
Can we remove the word `Timeout` from here please? A 429 isn't a timeout, its the client making requests too quickly - we can probably just go with `AssertionError: Unexpected 429`.